src/generic_ui/scripts/ui.ts

```diff
@@ -588,6 +588,10 @@ export class UserInterface implements ui_constants.UiApi {
       } else {
         // Old v1 invites are base64 encoded JSON
         var token = invite.substr(invite.lastIndexOf('/') + 1);
+        if (token[0] === '#') {
```

Comment:
Let's add a unit test to `ui.spec.ts` to help ensure pre-fragment URLs continue work - there are already so many invite formats!